app/src/main/res/values/strings.xml

```diff
@@ -5,16 +5,33 @@
     <string name="nav_user_title">Mine</string>
     <string name="setting_chinese">CN</string>
     <string name="setting_english">EN</string>
+    <string name="change_zh_message">App language changed to Chinese</string>
+    <string name="change_en_message">App language changed to English</string>
     <string name="user_screen_title" translatable="false">COMING SOON...</string>
 
     <!-- Anime Detail Screen -->
     <string name="intro_header_title">About</string>
     <string name="anime_type_title">Type: %1$s</string>
     <string name="anime_country_title">Country: %1$s</string>
-    <string name="anime_year_title">Published Year: %1$s</string>
+    <string name="anime_release_year_title">Published Year: %1$s</string>
     <string name="anime_status_title">Status: %1$s</string>
     <string name="maxim_header_title">Maxim</string>
     <string name="episode_header_title">Favourite Episodes</string>
+    <string name="empty_description_message">We need you input some more information for this anime …</string>
```

Comment:
remove space and replace to `.`: `We need you to provide more information about this anime...`